twistedchecker/functionaltests/docstring_pass.py

```diff
@@ -87,6 +89,11 @@ def otherProperty(self, value):
         pass
 
 
+    @aliasForProperty.setter
+    def decorated(self):
```

Comment:
can you please add a comment or docstring here to document what test scenario is covered by this example?